<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Metod1</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="createButton">
             <property name="text">
              <string>CreateJsonFile</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="readButton">
             <property name="text">
              <string>ReadJsonFile</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="writeButton">
             <property name="text">
              <string>WriteJsonFile</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Metod2</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPushButton" name="createButton_2">
             <property name="text">
              <string>CreateJsonFile</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="readButton_2">
             <property name="text">
              <string>ReadJsonFile</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="writeButton_2">
             <property name="text">
              <string>WriteJsonFile</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>290</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
